pkgs/development/libraries/frei0r/default.nix

```diff
@@ -1,26 +1,34 @@
-{ lib, stdenv, fetchurl, fetchpatch, cairo, cmake, opencv, pcre, pkg-config }:
+{ lib
+, config
+, stdenv
+, fetchurl
+, cairo
+, cmake
+, opencv
+, pcre
+, pkg-config
+, cudaSupport ? (config.cudaSupport or false) && stdenv.hostPlatform.isx86_64
```

Comment:
Why only x86_64? A comment would be nice.

Reply:
Hmm I have just copied this part from a different package. Just read though, that CUDA has support for ARM as well. I'm not sure about the support in NixOS though. I can remove the platform check, it isn't used in many packages it seems.